walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,80 @@
 
 package com.google.sps;
 
+import java.util.Arrays;
```

Comment:
Question: Do you need this? Can you use ArrayList.asList instead? (With my comment below you might not even need to call the asList() method)